it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/IOStressTestBase.java

```diff
@@ -101,12 +111,16 @@ public MultiplierDoFn(int startMultiplier, List<LoadPeriod> loadPeriods) {
       this.loadPeriods = loadPeriods;
     }
 
+    @Setup
+    public void start() {
+      this.startTimesMillis = Instant.now().getMillis();
+    }
+
     @DoFn.ProcessElement
-    public void processElement(
-        @Element T element, OutputReceiver<T> outputReceiver, @DoFn.Timestamp Instant timestamp) {
+    public void processElement(@Element T element, OutputReceiver<T> outputReceiver) {
 
       int multiplier = this.startMultiplier;
-      long elapsedTimeMillis = timestamp.getMillis() - startTimesMillis;
+      long elapsedTimeMillis = Instant.now().getMillis() - startTimesMillis;
```

Comment:
What are you trying to accomplish here? One side effect of this change is that now 2 different load tests will send different volumes of messages (e.g. if I'm slow processing messages, I'll get extra messages because more of them will have late timestamps).

I'm not super familiar with this code, so possible I'm misunderstanding the previous behavior, but growing load like this seems like it could be suboptimal